packages/terra-site/src/examples/image/ImageDefault.jsx

```diff
@@ -19,6 +19,9 @@ const ImageDefault = () => (
     <br />
     <h2>Fluid</h2>
     <Image src={placeholderPic1000x200} alt="fluid image" isFluid />
+    <br />
+    <h2>Placeholder</h2>
+    <Image src={placeholderPic150x150} placeholderSrc={placeholderPic1000x200} alt="placeholder image" />
```

Comment:
This example makes it hard to test. Can we use a completely different image to test with?

Reply:
What do you mean? The src and placeholderSrc are different images

Reply:
The main issue is that it's the same as the fluid image, and it's making it difficult for our solution designer to test that the placeholder is working. 

I suggest making it a completely different image that isn't the space image we use for the rest of the images on that page.